main.js

```diff
@@ -229,6 +235,24 @@ function updateMassLeads(leadIds, inputLead, masterLeads, massEditLead) {
 
 // LEAD DATA FUNCTIONS
 
+function sortLeads(leads, prop) {
+  var leadsCopy = leads;
```

Comment:
This isn't a real copy. Objects (including Arrays and Functions) are pass by reference. `Array.prototype.slice()` can be used to make a shallow copy.